src/docs/010_manual/20_install.adoc

```diff
@@ -58,20 +72,15 @@ NOTE: `dtcw.bat` wraps the `dtcw.ps1` script and executes it in powershell. This
 
 NOTE: In order to support development for your project on both, Linux and Windows, it makes sense to download all three scripts to your project.
 
-NOTE: if you use `cmd.exe`, please follow the instructions for powershell in the manual. We will not further differentiate between both.
+NOTE: if you use `cmd.exe`, please refer and follow the instructions for powershell. We will not further differentiate between both. You should however replace `.ps1` with `.bat`.
 
```

Comment:
Line 75

NOTE: If you use cmd.exe, follow the PowerShell instructions but replace `.ps1` with `.bat`.

**@Bennykillua Look for ways to really simplify the phrasing right down. Also, remember correct casing (PowerShell, not powershell). I have also started a standard where text following TIP: or NOTE: starts with a capital letter.**